### PR TITLE
PSGS-926: native-asset-scenarios

### PR DESCRIPTION
**Pull request by chuchulo**

### CODE REVIEW

File: examples/java/native-assets-example/src/main/java/iog/psg/assets/Scenarios.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                .get();
+
+
+        Map<String, Value> data = new HashMap<>();
+        Struct struct = com.google.protobuf.Struct.newBuilder()
+                .putAllFields(data)
+                .build();
+        //Mint `100` tokens of the native asset defined with `nativeAssetId`
+        CompletableFuture<NativeAssetTransaction> mintResult = new CompletableFuture<>();
+        api.mintAssetWithArbitraryMetadata(nativeAssetId, 100L, null, 4, lastResult(mintResult, r -> r.getAssetTx()));
```
Comment (chuchulo) [inline examples/java/native-assets-example/src/main/java/iog/psg/assets/Scenarios.java:96]: TODO replace the null with proper struct, once this fix is in place https://github.com/input-output-hk/psg-services/pull/351

File: examples/java/native-assets-example/src/main/java/iog/psg/assets/Scenarios.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+            @Override
+            public void onCompleted() {
+                result.complete(buff.get(buff.size() - 1));
+            }
+        };
+
+    }
+
+    public static void scenario1()  throws ExecutionException, InterruptedException, CborSerializationException  {
```
Comment (alexromanov) [inline examples/java/native-assets-example/src/main/java/iog/psg/assets/Scenarios.java:48]: At least we need to define scenario as a method name.

Comment (chuchulo) [reply examples/java/native-assets-example/src/main/java/iog/psg/assets/Scenarios.java]: renamed

**Comment by chuchulo** [comment]
reopened https://github.com/input-output-hk/PSG/pull/48

**Pull request closed without merging**